.prod/on_deploy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+python /app/manage.py migrate --noinput
+
+if [[ "$SEED_DEVELOPMENT_DATA" = "1" ]]; then
+    python /app/manage.py seed_data --development --superuser
```

Comment:
I think the `seed_data` management command is missing. Should we also add it to this PR?

Reply:
I removed it completely as we don't have the command and we don't need it, at least yet. I probably mixed this "built-in" command with other platform's command. 😄